Client/src/Pages/Register/index.jsx

```diff
@@ -150,7 +150,7 @@ const Register = () => {
         <div className="register-form-checks">
           <Check text="Must be at least 8 characters" />
           <div className="register-form-v-spacing-small"></div>
-          <Check text="Must contain one special character" />
+          <Check text="Must contain one special character, one number and one uppercase" />
```

Comment:
This should be in a separate pull request as it is resoving a different issue.

The rule of thumb is one issue per pull request, otherwise if we have to revert the merge for one of the issues we have to revert both of them.